File: source/frontend/projects/pulse/src/app/api/models/active-session-details-dto.ts
/* tslint:disable */
/* eslint-disable */
import { SessionCheckinDetailsDto } from './session-checkin-details-dto';
import { SessionDetailsDto } from './session-details-dto';
import { SessionEmoticonDetailsDto } from './session-emoticon-details-dto';
export type ActiveSessionDetailsDto = SessionDetailsDto & {
'emoticons': Array<SessionEmoticonDetailsDto>;
'activeCheckin'?: SessionCheckinDetailsDto | null;
};

File: source/backend/projects/Pulse.Api/Models/Sessions/ActiveSessionDetailsDto.cs
namespace Pulse.Api.Models.Sessions
{
    using System.ComponentModel.DataAnnotations;

    public class ActiveSessionDetailsDto : SessionDetailsDto
    {
        [Required]
        public SessionEmoticonDetailsDto[] Emoticons { get; set; }

        public SessionCheckinDetailsDto ActiveCheckin { get; set; }
    }
}
